x-pack/package.json

```diff
@@ -65,6 +65,7 @@
     "@types/json-stable-stringify": "^1.0.32",
     "@types/jsonwebtoken": "^7.2.7",
     "@types/lodash": "^3.10.1",
+    "@types/memoize-one": "^4.1.0",
```

Comment:
Should this type def version match that of the main dependency?

`"memoize-one": "^5.0.0",`

Reply:
The latest from here: https://www.npmjs.com/package/@types/memoize-one

Is `4.1.1` which the carrot will catch so there is no `5.0.0` at this time.